src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/GroupCollection.cs

```diff
@@ -210,7 +211,9 @@ void IList.RemoveAt(int index)
             return new Enumerator(this);
         }
 
-        public bool TryGetValue(string key, out Group value)
+#pragma warning disable CS8614 // Nullability of reference types in type of parameter doesn't match implicitly implemented member.
+        public bool TryGetValue(string key, [NotNullWhen(true)] out Group? value)
+#pragma warning restore CS8614 // Nullability of reference types in type of parameter doesn't match implicitly implemented member.
```

Comment:
Nit: Just need one comment at the opening #pragma.

Reply:
fixed